projectq/ops/_gates_test.py

```diff
@@ -91,6 +99,14 @@ def test_swap_gate():
     assert isinstance(_gates.Swap, _gates.SwapGate)
 
 
+def test_sqrtswap_gate():
+    gate = _gates.SqrtSwapGate()
+    gate_sq = _gates.SwapGate()
+    assert str(gate) == "SqrtSwap"
+    assert np.array_equal(gate.matrix * gate.matrix, gate_sq.matrix)
```

Comment:
This is a useful test 👍 
Can you also add a trivial test checking the gate matrix? As there are always two different roots, we might want to make sure we don't change convention of which one we pick